src/libraries/System.Reflection.Context/Directory.Build.props

```diff
@@ -1,7 +1,10 @@
 ﻿<Project>
   <Import Project="..\Directory.Build.props" />
   <PropertyGroup>
-    <AssemblyVersion>4.0.3.0</AssemblyVersion>
+    <!-- this assembly is inbox in desktop, do not version it unless you
+         plan on shipping a new desktop version out of band. Instead add API
+         to a different assembly. -->
+    <AssemblyVersion Condition="'$(TargetFramework)' == 'netstandard2.0'">4.0.3.0</AssemblyVersion>
```

Comment:
Why is this not "4.0.0.0"? Does "4.0.3.0" mean we serviced it in the past?

Reply:
resolving this, since that conversation is happening above in https://github.com/dotnet/runtime/pull/84079#discussion_r1152273413.